### DISCUSSION TITLE
ImportError: cannot import name 'make_generic_validator' from 'pydantic.class_validators'

### DISCUSSION THREAD

**Discussion by tt053626** [open-sdg/open-sdg#2019] [Q&A] [answered]
We had a new error message when updating meta-data in the data-repository since yesterday:

_"ImportError: cannot import name 'make_generic_validator' from 'pydantic.class_validators' (/opt/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages/pydantic/class_validators.py)"_

Hier is a screen shot from github-actions:
![error](https://github.com/open-sdg/open-sdg/assets/79691995/80b6a6cc-00db-41a7-aca4-d54e88cabc7e)

Thanks for helping!

**Comment by sarah-beasley** [comment]
Hi, thanks for the message, this seems to be affecting most Open SDG users at the moment! Just waiting for our developer @brockfanning to announce the best fix for this which shouldn't be too long!

**Comment by christoph-LSN** [comment, marked as answer]
@tt053626 have you tried this: #2017

**Reply by tt053626** [reply]
That works, thanks a lot!
